src/main/java/com/auth0/client/auth/AuthAPI.java

```diff
@@ -1359,7 +1359,7 @@ public Builder withHttpClient(Auth0HttpClient httpClient) {
          */
         public AuthAPI build() {
             return new AuthAPI(domain, clientId, clientSecret, clientAssertionSigner,
-                Objects.nonNull(httpClient) ? httpClient : DefaultHttpClient.newBuilder().build());
+                Objects.nonNull(httpClient) ? httpClient : DefaultHttpClient.newBuilder().withMaxRetries(0).build());
```

Comment:
don't retry auth requests on 429 by default

Reply:
Interesting on why we are not retrying for Auth requests by default

Reply:
I reverted this change in a new commit; we can talk about this more.